src/effects/backends/effectmanifest.cpp

```diff
@@ -0,0 +1,25 @@
+#include "effects/backends/effectmanifest.h"
+
+QDebug operator<<(QDebug dbg, const EffectManifest& manifest) {
+    return dbg.maybeSpace() << QString("EffectManifest(%1)").arg(manifest.id());
+}
+
+bool EffectManifest::hasMetaKnobLinking() const {
+    for (const auto& pParameterManifest : m_parameters) {
+        if (pParameterManifest->defaultLinkType() !=
+                EffectManifestParameter::LinkType::NONE) {
+            return true;
+        }
+    }
+    return false;
+}
+
+bool EffectManifest::alphabetize(
+        EffectManifestPointer pManifest1, EffectManifestPointer pManifest2) {
+    // Sort built-in effects first before external plugins
+    int backendNameComparision = static_cast<int>(pManifest1->backendType()) -
```

Comment:
this is brittle because it relies on a specific relationship between the backend types.  Can you make it more explicit and not dependent on the int relationships? (what if we add another backend type?).  OR, add documentation to backendtype to explain that it affects sort order